Description: The MySQL C API starting with MySQL 8.0. The my_bool type was removed in MySQL 8.0 and later versions

---
 gwlib/dbpool_mysql.c |    2 --
 1 file changed,  2 deletions(-)

--- a/gwlib/dbpool_mysql.c	
+++ b/gwlib/dbpool_mysql.c	(date 1727794907249)
@@ -245,7 +245,7 @@
         case MYSQL_TYPE_TIMESTAMP:
             bind[i].buffer_type = field->type;
             bind[i].buffer = (char*)gw_malloc(sizeof(MYSQL_TIME));
-            bind[i].is_null = gw_malloc(sizeof(my_bool));
+            bind[i].is_null = gw_malloc(sizeof(bool));
             bind[i].length = gw_malloc(sizeof(unsigned long));
             break;
         default:
@@ -253,7 +253,7 @@
             bind[i].buffer = gw_malloc(field->length);
             bind[i].buffer_length = field->length;
             bind[i].length = gw_malloc(sizeof(unsigned long));
-            bind[i].is_null = gw_malloc(sizeof(my_bool));
+            bind[i].is_null = gw_malloc(sizeof(bool));
             break;
         }
     }
